projects/vdk-heartbeat/src/vdk/internal/heartbeat/job_controller.py

```diff
@@ -336,6 +336,31 @@ def start_job_execution(self):
         ), "Failed to start data job execution."
         log.info("Execution started successfully.")
 
+    @LogDecorator(log)
+    def wait_job_execution_started(self):
+        log.info("Checking if data job execution is still running.")
+        start = time.time()
+
+        while time.time() - start < self.config.RUN_TEST_TIMEOUT_SECONDS:
+            response = self._execute(
+                [
+                    "execute",
+                    "--list",
+                    "-t",
+                    self.config.job_team,
+                    "-n",
+                    self.config.job_name,
+                    "-o" "json",
+                ]
+                + self.__get_rest_api_url_arg()
+            )
+            execution_list = json.loads(response)
+
+            if execution_list:
+                return
+            time.sleep(10)
+        raise Exception("Job never started")
```

Comment:
Isn't there a better exception class to raise here? Something native to vdk perhaps.

Reply:
also maybe include some identifying information from the config object in the exception message, it looks a bit bare to me right now

Reply:
what exception would you suggest? 
I added job name. 
Let me know if you think it looks good enough?

Reply:
Looking through the vdk-heartbeat codebase it seems no vdk-specific exception classes are used. In that case it's probably fine to keep it as Exception.